geomesa-tools/src/main/scala/org/locationtech/geomesa/tools/export/ExportParams.scala

```diff
@@ -20,7 +20,7 @@ trait FileExportParams {
   @Parameter(names = Array("--gzip"), description = "Level of gzip compression to apply to output, from 1-9")
   var gzip: Integer = null
 
-  @Parameter(names = Array("-F", "--output-Format"), description = "File format of output files (csv|tsv|gml|json|shp|avro)")
+  @Parameter(names = Array("-F", "--output-format"), description = "File format of output files (csv|tsv|gml|json|shp|avro)")
```

Comment:
good point - not sure how this ended up capitalized in the first place - all the long-form arguments should be lower case